restful.go

```diff
@@ -26,15 +26,15 @@ type APIClient struct {
 
 	ApiEndpoint string
 	Host        string
-
-	User     string
-	Password string
-
+	User        string
+	Password    string
 	AccessToken string
+	Tenant      string
+	Warehouse   string
 
-	Tenant    string
-	Warehouse string
-
+	WaitTimeSecs         time.Duration
```

Comment:
I think it is in integer unit as it counts based on seconds

Reply:
In DSN, it used like this `"databend+https://username:password@tn3ftqihs.ch.aws-us-east-2.default.databend.com/test?timeout=1s&wait_time_secs=10s`

Reply:
It seems wired with both `_secs` in name and `s` in value